rules-tests/Php85/Rector/FuncCall/OrdSingleByteRector/Fixture/multibyte_str.php.inc

```diff
@@ -0,0 +1,9 @@
+<?php 
+namespace Rector\Tests\Php85\Rector\FuncCall\OrdSingleByteRector\Fixture;
+echo ord('abc');
```

Comment:
please add test for:

```php
$var = 'abc';
echo ord($var);
```

which should be changed to:

```diff
$var = 'abc';
-echo ord($var);
+echo ord($var[0]);
```

or, just skip if passed argument is not `Node\Scalar\Int_` and `Node\Scalar\String_`

Reply:
Ok